data/en/cffunction.json

```diff
@@ -6,21 +6,21 @@
 	"params": [
-		{"name":"name","description":"A string; a component method that is used within the\n cfcomponent tag.","required":true,"default":"","type":"string","values":[]},
-		{"name":"returntype","description":"String; a type name; data type of the function return value\n Required for a web service; Optional, otherwise.","required":false,"default":"any","type":"string","values":["any","array","binary","boolean","date","guid","numeric","query","string","struct","UUID","variablename","void","xml","(component name)"]},
-		{"name":"roles","description":"A comma-delimited list of CFML security roles that\n can invoke the method. Only users who are logged in with\n the specified roles can execute the function. If this\n attribute is omitted, all users can invoke the method","required":false,"default":"","type":"string","values":[]},
-		{"name":"access","description":"The client security context from which the method can be\n invoked:\n\n private: available only to the component that declares the\n method and any components that extend the component in\n which it is defined\n package: available only to the component that declares the\n method, components that extend the component, or any\n other components in the package\n public: available to a locally executing page or component\n method\n remote: available to a locally or remotely executing page\n or component method, or a remote client through a URL,\n Flash, or a web service. To publish the function as a\n web service, this option is required.","required":false,"default":"public","type":"string","values":["private","package","public","remote"]},
-		{"name":"output","description":"Specifies under which conditions the function can generate\n HTML output.","required":false,"default":"","type":"boolean","values":[true,false]},
-		{"name":"displayname","description":"Meaningful only for CFC method parameters. A value to be\n displayed in parentheses following the function name when\n using introspection to show information about the CFC","required":false,"default":"","type":"string","values":[]},
-		{"name":"hint","description":"Meaningful only for CFC method parameters. Text to be\n displayed when using introspection to show information\n about the CFC. The hint attribute value follows the syntax\n line in the function description","required":false,"default":"","type":"string","values":[]},
+		{"name":"name","description":"A string; a component method that is used within the cfcomponent tag.","required":true,"default":"","type":"string","values":[]},
+		{"name":"returntype","description":"String; a type name; data type of the function return value. Required for a web service; optional otherwise.\n","required":false,"default":"any","type":"string","values":["any","array","binary","boolean","date","guid","numeric","query","string","struct","UUID","(variable name)","void","xml","(component name)"]},
```

Comment:
`variablename` is a type keyword and should not be replaced with `(variable name)`